docs/source/quantization_weight_only.md

```diff
@@ -47,8 +47,17 @@ There are many excellent works for weight only quantization to improve its accur
 |  mse_range |      True     | Whether search for the best clip range from range [0.89, 1.0, 0.01] |
 |  n_blocks  |       5       |   Split the model into n blocks for AWQ search to avoid out-of-memory   |
 
+**GPTQ arguments**:
+|  gptq_args  | default value |                               comments                              |
+|:----------:|:-------------:|:-------------------------------------------------------------------:|
+| actorder | False |   Whether to sort Hessian's diagonal values to rearrange channel-wise quantization order|
+|  percdamp | 0.01 | Add a small Hessian's diagonal values to itself, which increase Hessian's numerical stability|
```

Comment:
what does percdamp mean? damp?

Reply:
added more explanation in quantization_weight_only.md